lib/stellar_core_commander/process.rb

```diff
@@ -72,6 +103,22 @@ def close_ledger
       true
     end
 
+    Contract None => Hash
+    def metrics
+        response = server.get("/metrics") rescue false
+        if response
+          body = ActiveSupport::JSON.decode(response.body)
+          body["metrics"]
+        else
+          {}
+        end
+    end
```

Comment:
I often see ruby code, and it is my own personal style as well (although I would probably add type filters onto my rescue), that would express this same function as:

```ruby
def metrics
  server.get("/metrics")
  body = ActiveSupport::JSON.decode(response.body)
  body["metrics"]
rescue
  {}
end
```



A ruby `def` scope can have a `rescue` clause bolted onto it as though it were a `begin` block.  
I personally find it tough to see one-line `rescue` addendum and try to avoid them, but that's just 
personal style.  This method here is just fine, I'm leaving this comment more for the sake of 
explaining intricacies of ruby :)